functions/uptime/index.js

```diff
@@ -124,57 +124,39 @@ function deleteData() {
     });
 }
 
+/**
+ * @param {string} userIP
+ */
 function getGeoLocationInfo(userIP) {
```

Comment:
The implementation is the same as for the `usage` function, just without adding the `org` to the returned object